### PR TITLE
Feature/solr search implementation

### PR DESCRIPTION
**Pull request by robert-arias**
# Solr implementation and related content view more fix

## Jira issue:

- None.

## How to test

- [ ] Search with solr.
- [ ] Open a news article, go to the related news block on the bottom and click on "Ver más".

### CODE REVIEW

**Comment by rigoucr** [comment]
Hey Robert looking it your PR  I noticed that you forgot to add which commands did I needed to replicated your functionality in my local env. We always add the commands that the reviewer will require to setup the env , in this case the required ones were: `composer install && chirripo drush -- cim -y && chirripo drush -- cr`

When I tried to setup your branch in my local env, the `chirripo drush cim -y` command didn't work as expected:
```
 // Import the listed configuration changes?: yes.

 [error]  Drupal\Core\Config\ConfigImporterException: There were errors validating the config synchronization.
Unable to install the <em class="placeholder">search_api_autocomplete</em> module since it does not exist. in Drupal\Core\Config\ConfigImporter->validate() (line 755 of /var/www/html/web/core/lib/Drupal/Core/Config/ConfigImporter.php).

In ConfigImportCommands.php line 357:

  The import failed due to the following reasons:
  Unable to install the <em class="placeholder">search_api_autocomplete</em> module since it does not exist.
```

Seems like you forgot to add the module search_api_autocomplete to the composer.json

**Comment by robert-arias** [comment]
Thank you for the feedback! Yes, I completely forgot to add the required commands for a smooth installation on your local dev; won't happen again!

About the error: I don't know why that error is being triggered, because the module is in the composer.json [here](https://github.com/robert-arias/news-website/blob/3b682fc79d19ebf707fed788e68b1f31bfb11224/composer.json#L58).

**Comment by robert-arias** [comment]
The installation error has been fixed!

File: config/sync/core.extension.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ module:
   datetime: 0
   datetime_range: 0
   dblog: 0
+  devel: 0
```
Comment (rigoucr) [inline config/sync/core.extension.yml:20]: This module belongs only to the dev config context
```suggestion

```

File: config/sync/core.extension.yml [hunk 2]
```diff
@@ -49,8 +51,12 @@ module:
   rdf: 0
   responsive_image: 0
   retina_images: 0
+  search_api: 0
+  search_api_autocomplete: 0
+  search_api_solr: 0
   seckit: 0
   shs: 0
+  stage_file_proxy: 0
```
Comment (rigoucr) [inline config/sync/core.extension.yml:59]: This module belongs only to the dev config context
```suggestion

```

File: config/sync/core.extension.yml [hunk 3]
```diff
@@ -61,6 +67,7 @@ module:
   twig_tweak: 0
   update: 0
   user: 0
+  views_ui: 0
```
Comment (rigoucr) [inline config/sync/core.extension.yml:70]: This module belongs only to the dev config context
```suggestion

```

**Review by rigoucr — CHANGES_REQUESTED** [review]
almost there

**Review by rigoucr — APPROVED** [review]
Ship it :shipit:  !! thx!!!

**Pull request merged**